src/tr/eldermanga/src/eu/kanade/tachiyomi/extension/tr/eldermanga/ElderManga.kt

```diff
@@ -0,0 +1,237 @@
+            }
+        }
+
+        return MangasPage(mangas, false)
+    }
+
+    override fun fetchSearchManga(
+        page: Int,
+        query: String,
+        filters: FilterList,
+    ): Observable<MangasPage> {
+        if (query.startsWith(URL_SEARCH_PREFIX)) {
+            val url = "$baseUrl/manga/${query.substringAfter(URL_SEARCH_PREFIX)}"
+            return client.newCall(GET(url, headers)).asObservableSuccess().map { response ->
+                val document = response.asJsoup()
+                when {
+                    isMangaPage(document) -> MangasPage(listOf(mangaDetailsParse(document)), false)
+                    else -> MangasPage(emptyList(), false)
+                }
+            }
+        }
+        return super.fetchSearchManga(page, query, filters)
+    }
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val url = "$CDN_URL/series/search/navbar".toHttpUrl().newBuilder()
+            .addQueryParameter("search", query)
+            .build()
+        return GET(url, headers)
+    }
+
+    override fun searchMangaParse(response: Response): MangasPage {
+        val dto = json.decodeFromString<List<SearchDto>>(response.body.string())
+        val mangas = dto.map {
+            SManga.create().apply {
+                title = it.name
+                thumbnail_url = CDN_URL + it.image
+                url = "/manga/${it.id}/${title.lowercase().trim().replace(" ", "-")}"
+            }
+        }
+
+        return MangasPage(mangas, false)
+    }
+
+    override fun searchMangaFromElement(element: Element) = throw UnsupportedOperationException()
+
+    override fun searchMangaNextPageSelector() = throw UnsupportedOperationException()
+
+    override fun searchMangaSelector() = throw UnsupportedOperationException()
+
+    override fun mangaDetailsParse(document: Document) = SManga.create().apply {
+        with(document.selectFirst("#content")!!) {
+            title = selectFirst("h1")!!.text()
+            thumbnail_url = selectFirst("img")?.absUrl("src")
+            genre = select("a[href^='search?categories']").joinToString { it.text() }
+            description = selectFirst("div.grid h2 + p")?.text()
+            val pageStatus = selectFirst("span:contains(Durum) + span")?.text() ?: ""
+            status = when {
+                pageStatus.contains("Devam Ediyor", "Birakildi") -> SManga.ONGOING
+                pageStatus.contains("Tamamlandi") -> SManga.COMPLETED
+                pageStatus.contains("Ara Veridi") -> SManga.ON_HIATUS
+                else -> SManga.UNKNOWN
+            }
+
+            setUrlWithoutDomain(document.location())
+        }
+    }
+
+    override fun chapterListSelector() = "div.list-episode a"
+
+    override fun chapterFromElement(element: Element) = SChapter.create().apply {
+        name = element.selectFirst("h3")!!.text()
+        date_upload = element.selectFirst("span")?.text()?.toDate() ?: 0L
```

Comment:
```suggestion
        date_upload = dateFormat.tryParse(element.selectFirst("span")?.text())
```
import `keiyoushi.utils.tryParse`